include/class.config.php

```diff
@@ -77,7 +77,11 @@ function get($key, $default=null) {
     }
 
     function exists($key) {
-        return $this->get($key, null) ? true : false;
+        if ((isset($this->session) && array_key_exists($key, $this->session))
+                || array_key_exists($key, $this->config))
+            return true;
+
```

Comment:
While this somehow addresses the issue I think it's the wrong approach 1) Doesn't consider defaults & Duplicates the code etc. 

I think simply checking for the returned default is sufficient.

```php
return  ($this->get($key, null) !== null);
```